VTTiny/Plugins/PluginManager.cs

```diff
@@ -25,21 +25,40 @@ internal static class PluginManager
         /// </summary>
         public static void LoadAllPlugins()
         {
+            Console.WriteLine(PLUGINS_FOLDER_PATH);
             _plugins = new();
-
-            foreach (var directory in Directory.GetDirectories(PLUGINS_FOLDER_PATH))
+            if (!Directory.Exists(PLUGINS_FOLDER_PATH))
+                Directory.CreateDirectory(PLUGINS_FOLDER_PATH);
+            //First boot after download will not have the plugins folder, and crash
+            
+            foreach (var file in Directory.GetFiles(PLUGINS_FOLDER_PATH)) // Apparently this was all directories, not files.
             {
-                var plugin = Plugin.TryLoadFromFile(directory);
+                Console.WriteLine(file);
+                PluginLoadContext context = new(file);
+                var assem = context.LoadFromAssemblyName(new AssemblyName(Path.GetFileNameWithoutExtension(file)));
```

Comment:
Why are we loading the plugin twice here?

Reply:
still finding a good solution for this one, i'm debating between using Plugin framework, or a nuget package for loading it, right now (since it doesn't really do anything because there is no plugin's yet) it does nothing